utils/tinystr/src/error.rs

```diff
@@ -4,6 +4,9 @@
 
 use displaydoc::Display;
 
+#[cfg(feature = "std")]
```

Comment:
Issue: it needs to be added to tinystr's Cargo.toml, too

Reply:
Ah forgot to check that in. Late night changes :roll_eyes:

Reply:
Missing some feature propagation it seems...